test/unit/query/builder.js

```diff
@@ -134,7 +134,17 @@ describe("QueryBuilder", function() {
     });
   });
 
-  it("basic alias", function() {
+  it("basic select with alias as property-value pairs", function() {
+    testsql(qb().select({bar: 'foo'}).from('users'), {
```

Comment:
should test also `.select('colName', { bar: 'foo'})` and `.select(['colName', { bar: 'foo'}])`

Reply:
Sure, you're right.